installation/install.rst

```diff
@@ -64,33 +66,59 @@ You can find user stories and example builds on our
 2. Yet another recommendation used by some community members is HP Proliant
    Microserver Gen8.
 
+3. Raspberry Pi4 and RPi 400 have also both been reported as working as
+   intended via our newer Rockstor 4 "Build on openSUSE" variant. N.B. RPi 400
+   requires at least a 15.3 profile for the internal keyboard to work.
+
 Upgrading Rockstor
 ------------------
-Rockstor is under continuous development and we push tested updates in small
-batches. These updates are easy to install. While we roll-out major releases
-that require complete OS install, we try to make these releases as infrequent
-as possible. We strive to make most updates non-disruptive.
+Rockstor is under continuous development and we generally release updates in
+small batches. These updates are easy to install. But we do roll-out major
+releases that require a complete re-installs. Upgrading from Rockstor 3 (CentOS
+based) to Rockstor 4 "Built on openSUSE" is one such update.
+But such updates are very rare.
+
+Non re-install Rockstor updates can be installed in two ways :
+
+1. Install updates from the Web-UI (recommended):
+On the Rockstor Web-UI, far top-right, you will see an upward facing arrow if
+any Rockstor package updates are available. All upstream packages except the
+'rockstor' package can similarly be installed by clicking on a flashing
+wifi-like icon. Again this is not shown if no updates are available. But it
```

Comment:
This also seems like a good spot for a screenshot of the "wifi-like" icon.
Additional minor edit suggestion:
```suggestion
wifi-like icon. Again, this is not shown if no update is available, but it
```

Reply:
OK. Again I'll add via a commit as merging here might be tricky given I've now also added a pic in here via other changes. I may be in for a merge conflict with all these in GitHub and outside of GitHub changes on the same file. Oh well.